src/theme/themeProperties.ts

```diff
@@ -38,7 +37,6 @@ const themeProperties = {
     primaryFontColor: '#2f3336',
     dividerColor: '#d5d5d5',
     secondaryColor: '#ecb641',
-    backgroundColor: '#fff',
```

Comment:
Don't remove `backgroundColor` - it may differ from `light` color, and serves a different purpose, even though the same color is currently used.